portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,7 +63,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    //recieves submitted comment 
+    //Recieves submitted comment 
```

Comment:
Receives

Reply:
changed

Reply:
Sorry `Receives` has been misspelled as `Recieves`, didn't mean for this much back and forth over a typo but that was what I meant to capture with my comment :)

Reply:
my bad!! just changed it